<launch>

  <node pkg="event_based_rgbd_ros_driver" type="publisher" name="publisher" respawn="false" output="screen">
    <param name="camera_name"		value="Gen3_VGA" />
    <param name="exposure_time"		value="235" />
    <param name="biases_addr"		value="../catkin_ws/src/event_based_rgbd_ros/event_based_rgbd_ros_driver/cfg/bias_sensitive.bias"/>

    <param name="roi_width"		value="60" />
    <param name="roi_height"	value="40" />
    <param name="roi_x"		value="300" />
    <param name="roi_y"		value="200" />
    <param name="region_flag"	value="false" />

  </node>

</launch>
